teamapps-ux/src/main/java/org/teamapps/ux/session/SessionContext.java

```diff
@@ -72,6 +72,18 @@
 
 import static org.teamapps.common.util.ExceptionUtil.softenExceptions;
 
+/**
+ * Provides access to several aspects of a Ui-Session and allows sending commands and updates to the client.
+ * A UI-session starts upon opening the website and ends when leaving or reloading it. The Communication between client
+ * and server happens via websocket.
+ * Only one thread can interact with a Ui-Session at a time. Therefore, all interactions with the session context from
+ * outside must be done using the {@link #runWithContext(Runnable)} method.
```

Comment:
You might also mention that
- events coming from the client are generally handled in a thread bound to the UI Session.